dipy/boots/resampling.py

```diff
@@ -160,14 +170,15 @@ def __calc_z0(x, p_0, statistic, eps, a_hat, sigma_hat):
         b_hat[i] = __tt_dot_dot(i, x, p_0, statistic, eps)
         tt_dot[i] = __tt_dot(i, x, p_0, statistic, eps)
     b_hat = b_hat / (2 * n**2)
-    c_q_hat = (__tt(x, (1 - eps) * p_0 + eps * tt_dot / (n**2 * sigma_hat)
-                                                              , statistic) +
-               __tt(x, (1 - eps) * p_0 - eps * tt_dot / (n**2 * sigma_hat)
-                                                              , statistic) -
-               2 * __tt(x, p_0, statistic) ) / eps**2
+    c_q_hat = (__tt(x, (1 - eps) * p_0 + eps * tt_dot /
+                    (n**2 * sigma_hat), statistic) +
```

Comment:
For clarity, I would enclose the things that go together in parentheses:
     
    (__tt(x, ((1 - eps) * p_0) +
            (eps * tt_dot / (n**2 * sigma_hat)), statistic ...